### PR TITLE
fix(iOS): SimpleOrientationSensor Sample Crash

### PR DESCRIPTION
**Pull request by ajpinedam**
GitHub Issue (If applicable): closes https://github.com/unoplatform/uno/issues/17688

## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the current behavior?
Gallery iOS Application crashes when using the SimpleOrientationSensor sample page

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested on iOS.
- [ ] Tested on Wasm.
- [ ] Tested on Android.
- [ ] Tested on UWP.
- [ ] Tested in both **Light** and **Dark** themes.
- [x] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: Uno.Gallery/Views/SamplePages/SimpleOrientationSamplePage.xaml.cs [hunk 1]
```diff
@@ -1,84 +1,98 @@
-﻿using System;
```
Comment (kazo0) [inline Uno.Gallery/Views/SamplePages/SimpleOrientationSamplePage.xaml.cs:1]: What's the actual change that was done here? It seems to be that the whole file content was changed so it's not highlighting the specific change for the fix

**Review by MartinZikmund — APPROVED** [review]
When viewing change without whitespace the fix is easier to see. However, please check if the new formatting adheres to the rest of the project

File: Uno.Gallery/Views/SamplePages/SimpleOrientationSamplePage.xaml.cs (hunk 1 above)
Comment (Xiaoy312) [reply Uno.Gallery/Views/SamplePages/SimpleOrientationSamplePage.xaml.cs:1]: formatting change like this, while welcomed,
should be in its own PR or, the very least, its own commit

File: Uno.Gallery/Views/SamplePages/SimpleOrientationSamplePage.xaml.cs [hunk 2]
```diff
@@ -1,84 +1,98 @@
...
+using Windows.UI.Core;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+
+namespace Uno.Gallery.Views.Samples
+{
+	[SamplePage(
+		SampleCategory.NonUIFeatures,
+		"Simple Orientation",
+		Description = "This sensor detects the current quadrant orientation of the specified device as well as its face-up or face-down status.", DocumentationLink = "https://learn.microsoft.com/en-us/uwp/api/windows.devices.sensors.simpleorientationsensor")]
```
Comment (Xiaoy312) [inline Uno.Gallery/Views/SamplePages/SimpleOrientationSamplePage.xaml.cs:15]: if you were going to format this, dont go half way:
```suggestion
	[SamplePage(
		SampleCategory.NonUIFeatures,
		"Simple Orientation",
		Description = "This sensor detects the current quadrant orientation of the specified device as well as its face-up or face-down status.",
		DocumentationLink = "https://learn.microsoft.com/en-us/uwp/api/windows.devices.sensors.simpleorientationsensor")]
```

**Review by Xiaoy312 — APPROVED** [review]

File: Uno.Gallery/Views/SamplePages/SimpleOrientationSamplePage.xaml.cs (hunk 1 above)
Comment (ajpinedam) [reply Uno.Gallery/Views/SamplePages/SimpleOrientationSamplePage.xaml.cs:1]: It was not supposed to be a formatting change, some how Visual Studio Code made these changes. 

I am looking for a way to revert these space changes.

Comment (ajpinedam) [reply Uno.Gallery/Views/SamplePages/SimpleOrientationSamplePage.xaml.cs]: Fixed!

**Pull request merged**